.github/workflows/continuous-integration.yml

```diff
@@ -151,3 +151,65 @@ jobs:
 
       - name: "Run PHPUnit"
         run: "php bin/simple-phpunit -v"
+
+  phpunit_no_database:
```

Comment:
I'm not a big fan of running these new tests on all PR as it unnecessarily increases load on the CI. @nicosomb could we disable them by default (or drop the commit before merging)?

Reply:
Maybe we can just keep one? For MySQL or PG